rstcheck.py

```diff
@@ -688,6 +688,16 @@ def visit_literal_block(self, node):
 
         raise docutils.nodes.SkipNode
 
+    def visit_paragraph(self, node):
+        """Check syntax of reStructuredText."""
+        find = re.search(r'\[[^\]]*\]\([^\)]*\)', str(node))
+        try:
```

Comment:
Would this work okay without the `try/except`?

Reply:
@myint Thank you for the feedback! I've made some changes.
Regarding above, I used the try/except to catch *all* bad links in a file rather than just one, because this is a warning rather than an error, but I'm not sure?